webapp/sources/rudder/rudder-web/src/main/javascript/rudder/homePage.js

```diff
@@ -229,9 +243,10 @@ var complianceHoverColors =
   , "#DA291C" : "#ed1809ff"
   }
 
-function doughnutChart (id,data,count,colors,hoverColors) {
+function doughnutChart (id,data,colors,hoverColors) {
 
   var context = $("#"+id)
+  var count = data.values.length < 1 ? 0 : data.values.reduce((a, b) => a + b, 0);
```

Comment:
in place of count, compute the sum of values here. 
I didn't see a case where it should be ok to have a different count, and all calling place in scala backend use the same source of data to build both